From 51ea9a86b8c8115fb293c7b9c6bfd313fd3d5373 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Thu, 2 Aug 2012 15:59:35 +0800
Subject: ENGR00219184 HDMI AUDIO: set HDMI audio's SDMA priority to high

HDMI use SDMA for buffer switch. Set HDMI audio's SDMA priority to the
same level as other audio device. This adjust can avoid noise due to
SDMA cpu race failing.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 sound/soc/imx/imx-hdmi-dma.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/imx/imx-hdmi-dma.c b/sound/soc/imx/imx-hdmi-dma.c
index a743d10..443ff10 100644
--- a/sound/soc/imx/imx-hdmi-dma.c
+++ b/sound/soc/imx/imx-hdmi-dma.c
@@ -938,7 +938,7 @@ static int hdmi_sdma_alloc(struct imx_hdmi_dma_runtime_data *params)
 	dma_cap_mask_t mask;
 
 	params->dma_data.peripheral_type = IMX_DMATYPE_HDMI;
-	params->dma_data.priority = DMA_PRIO_MEDIUM;
+	params->dma_data.priority = DMA_PRIO_HIGH;
 	params->dma_data.dma_request = MX6Q_DMA_REQ_EXT_DMA_REQ_0;
 	params->dma_data.private = &params->sdma_params;
 
-- 
1.7.9.5

